docs/api/synthetics/monitors/add-monitor-api.asciidoc

```diff
@@ -0,0 +1,168 @@
+
+The request body should contain the attributes of the monitor you want to create. The required and default fields differ depending on the monitor type:
+
+*Common Fields*:
+
+`name`:: (Required, string): The monitor's name.
+
+`type`:: (Required, string): The monitor type (e.g., "http," "tcp," "icmp," or "browser").
+
+`schedule`:: (Required, object): The monitor's schedule.
+
+  * `number` (Optional, string, default: '3'): The number of intervals.
+
+  * `unit` (Optional, string, default: "minutes"): The unit of the schedule.
+
+:private-locations-url: https://www.elastic.co/guide/en/observability/current/synthetics-private-location.html
+:locations-command: https://www.elastic.co/guide/en/observability/current/synthetics-command-reference.html#elastic-synthetics-locations-command
+
+`locations`:: (https://github.com/elastic/synthetics/blob/main/src/locations/public-locations.ts#L28-L37[`Array<SyntheticsLocationsType>`])
+Where to deploy the monitor. Monitors can be deployed in multiple locations so that you can detect differences in availability and response times across those locations.
++
+To list available locations you can:
++
+  * Run the https://www.elastic.co/guide/en/observability/current/synthetics-command-reference.html#elastic-synthetics-locations-command[`elastic-synthetics locations`] command
+  with the {kib} URL for the deployment from which to fetch available locations.
+  * Go to *Synthetics* -> *Management* and click *Create monitor*.
+    Locations will be listed in _Locations_.
+
+`private_locations` (`Array<string>`)::
+
+The https://www.elastic.co/guide/en/observability/current/synthetics-private-location.html[Private Locations] to which the monitors will be deployed. These Private Locations refer to locations hosted and managed by you, whereas
+`locations` are hosted by Elastic. You can specify a Private Locations using the location's name.
++
+To list available Private Locations you can:
++
+  * Run the https://www.elastic.co/guide/en/observability/current/synthetics-command-reference.html#elastic-synthetics-locations-command[`elastic-synthetics locations`] command
+  with the {kib} URL for the deployment from which to fetch available locations.
+  * Go to *Synthetics* -> *Settings* and click *Private locations*.
+  Private Locations will be listed in the table.
+
+[NOTE]
+====
+You may provide either `locations` or `private_locations`, and you may also provide both if needed. At least one of them is required, but providing both is optional.
+====
+
+
+`enabled`:: (Optional, boolean, default: true): Whether the monitor is enabled.
+
+`tags`:: (Optional, array of strings): An array of tags. 
+
+`alert`:: (Optional, object, default: { status: { enabled: true }, tls: { enabled: true } }): Alert configuration.
+
+`service.name`:: (Optional, string): The APM service name.
+
+`timeout`:: (Optional, string, default: 16): The monitor timeout in seconds.
+
+`namespace`:: (Optional, string, default: "default"): The namespace.
```

Comment:
namespace has specific pattern requirements, they should probably be documented. How are we handling validation of that btw? Not 100% sure if we only do this currently on the front end, or also on the backend.

Makes me think if we're currently validating namespaces properly for project monitors, or if we ever got around to allowing custom namespaces for project monitors, which I believe was a still open ticket when I left.